src/api.spec.yaml

```diff
@@ -0,0 +1,595 @@
+---
+openapi: 3.0.2
+info:
+  title: Renku data sercice
+  description: A service that provides information about data, users and compute resources
+  version: v1
+servers:
+  - url: /api/data
+paths:
+  /resourcePools:
```

Comment:
@Panaetius the kg and notebooks already use a mix of underscores and camelcase. So I do not want to add a third variant. I would prefer to stick to camel case.

I can convert to underscores so that we have some more commonality between the different apis. But my problem with that is do I use underscores in response and requests bodies or not? I would prefer to use the same case across the board.

If I would convert to underscores I would then use underscores in uris, parameters, response and request bodies.

I also know the biggest users of this API will be the ui and there camelcase is definitely preferred.

Reply:
In that case it's fine. I didn't really know what other components use.